spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -0,0 +1,256 @@
+# Copyright (C) 2018 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.ssr
+description: Precise State Space Representation (SSR) corrections format
+stable: False
+public: True
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+
+ - OrbitClockHeader:
+    short_desc: Header for orbit/clock message.
+    desc: Header of a SSR orbit/clock correction message.
+    fields:
+        - time:
+            type: GPSTime
+            desc: GNSS reference time of the correction
+        - update_interval:
+            type: u8
+            units: s
+            desc: Update interval between consecutive corrections
+        - iod_ssr:
+            type: u8
+            desc: |
+                IOD of the SSR correction. A change of Issue Of Data
+                SSR is used to indicate a change in the SSR 
+                generating configuration
+        - n_corr:
```

Comment:
General question - do we want or need to pack content? Observations are packed because of bandwidth, but this pattern around size + index field is a little unwieldy. Are we sure we want to open up another use case for it?

Reply:
I think bandwidth is an important thin to optimize for this message thinking about how we may want use it as the corrections delivery to devices.  I don't see this being consumed very often (even less often than obs messages probably).

Reply:
@mark Yes I see your point, and that was definitely one of my question so thx for the feedback. 
The idea was that:
-  we dont want to use the corrections until we received all of them.
- There is a common data that would bandwidth if repeated on a single satellite basis
But I see the pain in term of encoding/decoding.
So for the first concern, there are other ways to reach that goal in the filter.  And for the bandwidth concern, I dont think we will send corrections like this to the user in the short term anyway (only used for the network engine interface), as the Skylark user correction message format is far from being defined at the moment.
So I would propose to switch to a static 1 message per satellite correction.
The rate is expected to be 1 orbit/clock/code message every 5s per satellite, so a burst of 3 * (32+24) messages every 5s for GPS+GLONASS. Sounds good?
What do you think about it?